roadmaps/skylark.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: contribute
+title: Skylark roadmap
+---
+
+# Skylark Q3 2018 Roadmap
+
+*Last verified: July 18th, 2018*
```

Comment:
The European in me (which is the only "-an" in me) really wants this to be [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) (YYYY-MM-DD) date format, please.

Reply:
Me too :)
I copied the format from another doc, but I like YYYY-MM-DD much more.

Done